index.html

```diff
@@ -83,7 +83,7 @@ <h2 class='header'>Linux Instructions</h2>
                     <li>Download the .TAR.GZ file</li>
                     <li>Expand the file (`tar zxf servo-latest.tar.gz`)</li>
                     <li>`cd servo`</li>
-                    <li>Execute the `./runservo.sh` script to run Servo</li>
+                    <li>Execute the `./servo` script to run Servo</li>
```

Comment:
`./servo` isn't a script but the actual executable, so remove `the` and `script` from this line.